public/jsx/controls.jsx

```diff
@@ -97,6 +114,7 @@ class Controls extends React.Component {
               placeholder={config.startAddressPlaceholder}
             />
           </div>
+          <button onClick={this.setStartAddressToMyGeolocation.bind(this)}>Find Me</button>
```

Comment:
You can avoid needing to bind `this` by just using an arrow function on the method itself: `setStartAddressToMyGeolocation = () => {`

Reply:
Yeah, but I think we use some ancient version of something (browserify?) that complained about not supporting that syntax for declaring methods on a class 😱